src/components/MediaPlayer/VideoJSPlayer.js

```diff
@@ -222,7 +222,7 @@ function VideoJSPlayer({
    */
   React.useEffect(() => {
     if (isClicked && canvasIndex !== cIndex) {
-      switchPlayer(canvasIndex);
+      switchPlayer(canvasIndex, false);
```

Comment:
Why is `fromStart` false in this case?

Reply:
Because the current time of the player is set to the start value of the clicked structure item. By setting this flag to true, it will always set the time to the start of the new canvas within the function, which is the case for all the other cases except for when using structure navigation.